lib/src/auth.dart

```diff
@@ -119,47 +135,50 @@ abstract class Authenticator {
     if (!isValid) {
       refresh();
     }
-    http.Request request = new http.Request(type, path);
+    final request = new http.Request(type, path);
     if (body != null) {
       request.bodyFields = body;
     }
     final http.StreamedResponse response = await _client.send(request);
-    Map parsed = JSON.decode(await response.stream.bytesToString());
+    final parsed = JSON.decode(await response.stream.bytesToString());
     if (parsed.containsKey(kErrorKey)) {
       _throwAuthenticationError(parsed);
     }
     return parsed;
   }
 
-  // Requests the authentication token from Reddit based on parameters provided
-  // in [accountInfo] and [_grant].
+  /// Requests the authentication token from Reddit based on parameters provided
+  /// in [accountInfo] and [_grant].
   Future _requestToken(Map<String, String> accountInfo) async {
     // Retrieve the client ID and secret.
-    String clientId = _grant.identifier;
-    String clientSecret = _grant.secret;
+    final clientId = _grant.identifier;
+    final clientSecret = _grant.secret;
+    String userInfo = null;
```

Comment:
All declarations are implicitly `null`. There's a lint for this: http://dart-lang.github.io/linter/lints/avoid_init_to_null.html 😉